krita: update to 5.2.9

xtexx opened the pull request:
Topic Description
-----------------

krita: update to 5.2.9
- delete 3 patches that no longer needed.
- modify 0001-sip-6.8.patch to work with the current source code.

Package(s) Affected
-------------------

- krita: 5.2.9

Security Update?
----------------

No

Build Order
-----------

```
#buildit krita
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

ruaharico commented:
LTGM on amd64
![图片](https://github.com/user-attachments/assets/3c9a1a7f-ef68-406b-8bdd-417e3be701df)

ruaharico committed:
krita: update to 5.2.9
- delete 3 patches that no longer needed.
- modify 0001-sip-6.8.patch to work with the current source code.
